pkg/services/database/database.go

```diff
@@ -243,3 +243,23 @@ func (d *database) findPreloadWitLimitAndPageQuery(
 
 	return d.connectionRead.Table(table).Where(where).Limit(limit).Offset(page * limit)
 }
+
+// Deprecated: RawQuery starts a transaction and try to execute the raw query into database.
+// is not recommended using this and the method will not be available after cli v2.10.0
+func (d *database) RawQuery(rawQuery string) error {
```

Comment:
[Keep default format equals read interface](https://github.com/ZupIT/horusec-devkit/blob/cb1165154a4c15a69284a3cba604cbd31f4ca7ca/pkg/services/database/read_interface.go#L27)

In Write case entityPointer isn't necessary:
```suggestion
func (d *database) RawWrite(rawSQL string, values ...interface{}) response.IResponse {
```

And in the future we can change Read too:
```
func (d *database) RawRead(rawSQL string, entityPointer interface{}, values ...interface{})
```